task160_FinalTask/src/test/java/automationpractice/com/pages/ProductDetailsPage.java

```diff
@@ -0,0 +1,30 @@
+package automationpractice.com.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ProductDetailsPage extends BasePage {
+    @FindBy(id = "wishlist_button")
+    private WebElement ADD_TO_WISHLIST_BUTTON;
+    @FindBy(name = "Submit")
+    private WebElement ADD_TO_CART_BUTTON;
+    @FindBy(css = "#product_reference span")
+    private WebElement PRODUCT_ID;
+    @FindBy(css = "a.fancybox-close")
+    private WebElement ADDED_TO_WISHLIST_POPUP_CLOSE_BUTTON;
+
+    public ProductDetailsPage() {
+
+    }
```

Comment:
I think you can remove it

Reply:
I'm actually don't want to remove it, I'm checking if I on the expected page or not.

This check was missing, I've added it.